src/electionguard/ballot_compact.py

```diff
@@ -154,29 +155,30 @@ def _get_plaintext_contests(
     )
 
     contests: List[PlaintextBallotContest] = []
-    for manifest_contest in sorted(
-        internal_manifest.contests, key=lambda c: c.sequence_order
```

Comment:
Any location where there was a lambda sort it was exchanged to rely exclusively on the new `sequence_order_sort()`